pupil_src/shared_modules/audio_capture.py

```diff
@@ -152,6 +152,8 @@ def capture_thread(self, audio_src, running, recording):
                 in_container = av.open('none:{}'.format(audio_src), format="avfoundation")
             elif platform.system() == "Linux":
                 in_container = av.open('hw:{}'.format(audio_src), format="alsa")
+            elif platform.system() == "Windows":
+                in_container = av.open('audio={}'.format(audio_src), format="dshow", options={'audio_buffer_size':'23'})
```

Comment:
What is the reasoning to set this value to `23`? Is it possible that this value is device-dependent?

Reply:
No it's not device depdendent. It's the size of the capture buffer in miliseconds.